extensions/data-transfer/portability-data-transfer-apple/src/test/java/org/datatransferproject/datatransfer/apple/photos/AppleImporterTestBase.java

```diff
@@ -280,7 +281,8 @@ protected List<PhotoModel> createTestPhotos(@NotNull final int count) {
               ALBUM_DATAID_BASE + i,
               false,
                   null,
-                  new Date());
+                  new Date(),
```

Comment:
tl;dr maybe add test to ensure this line happens correctly over in `AppleMediaInterface`:

```java
    final PhotosProtocol.CreateMediaResponse createMediaResponse =
        createMedia(jobId.toString(), dataClass, newMediaRequestList);
```

 it easy enough to write an update to one of your unit tests to show this test-infra change gets utilized correctly and the outgoing API requests have the right state?

Reply:
> it easy enough to

typo: I mean to write _"*Is* it easy enough to ... ?"_

Reply:
Yeah I have added an assertion to check that newMediaRequest will have isFavorite set correctly.
extensions/data-transfer/portability-data-transfer-apple/src/test/java/org/datatransferproject/datatransfer/apple/photos/AppleMediaImporterTest.java

     ` assertThat(newMediaRequestList.stream().allMatch(newMediaRequest -> newMediaRequest.hasIsFavorite() && newMediaRequest.getIsFavorite()==1L)).isTrue();`

Reply:
nice!